mipengine/node/monetdb_interface/common_actions.py

```diff
@@ -23,7 +23,7 @@ def create_table_name(
     """
```

Comment:
Does the table name really need to be assembled inside the database interface? The db interface should just get the desired name for the table to create, the schema and just create it. It can check if the given name has characters that are not allowed but does it need to concatenate everything to assemble the table name inside there?

Reply:
This is not part of this PR, please make a ticket to discuss this.